components/compounds/Header.tsx

```diff
@@ -11,35 +11,38 @@ import { TranslatedText } from '../atoms/TranslatedText';
 
 const Menu = React.memo(() => {
   const router = useRouter();
-  const { locale } = useContext(TranslationContext);
+  const { locale, translation } = useContext(TranslationContext);
   // consider extracting to generic routing function
-  const handleButtonPress = (path: string) => {
-    router.push(pathFromComponents(locale, path));
+  const handleButtonPress = (path: string, hash?: string) => {
+    router.push({
+      pathname: pathFromComponents(locale, path),
+      hash: hash
+    });
   };
 
   return (
     <>
       <div className="hidden md:flex flex-row justify-between w-128">
         <LanguagePicker />
-        <Button variant="text" color="secondary" onClick={ () => handleButtonPress('/login') }>
-          Pricing
+        <Button variant="text" color="secondary" onClick={ () => handleButtonPress('/', 'pricing') }>
+          { translation.header.pricing }
         </Button>
         <Button variant="text" color="secondary" onClick={ () => handleButtonPress('/login') }>
-          Contact us
+          { translation.header.contactUs }
         </Button>
         <Button variant="contained" color="secondary" onClick={ () => handleButtonPress('/login') }>
-          Login
+          { translation.header.login }
```

Comment:
A bit tedious, but consider wrapping all text in `<TranslationText>` for consistency with line-break and html parsing